requirements.txt

```diff
@@ -6,7 +6,7 @@ flake8
 # https://github.com/pallets/markupsafe/issues/284
 markupsafe==2.0.1
 # https://github.com/pandas-dev/pandas/pull/45749
-pandas==1.1.5
+# pandas
```

Comment:
I don't think this is right, why are we removing pandas here?

Reply:
I think you're right about this; having an explicit version is a liability as we go to future versions of python but, until we're there happy to take this out.